content/en/docs/latest/ASS_Tags.md

```diff
@@ -574,9 +543,9 @@ The different `\k` tags create various effects:
 - `\ko`: Similar to `\k`, except that before highlight, the border/outline of
   the syllable is removed, and appears instantly when the syllable starts.
 
-_Note: There is an additional karaoke tag, `\kt`, which is very different from
-the other ones. It is rarely useful and Aegisub does not support that tag, so
```

Comment:
I think the "support" mentioned here refers to the various Aegisub-internal tools, such as the karaoke timing mode or the kanji timer. And probably the karaskel templater, too, since I haven't read anything there about this tag.

I didn't check this, though, because I didn't want to re-compile it.

Reply:
Yeah, it’s probably best to leave the note about lack of support (atm) in. Repushed with it added.

I don’t see how resetting times to something earlier can ever work with karaoke timing mode, but I guess using `\kt` to skip ahead and insert pauses between syllables (instead of using an empty/invisible syllable) does make sense.